### PR TITLE
Add es/jaeger tracing scenario

### PR DESCRIPTION
**Pull request by nhudson**
Adding in scenario to build EKS cluster and deploy Elasticsearch, Jaeger and run synthetic load tests on the stack.

This is mainly taken from @mathisve https://github.com/mathisve/o11y-bench repo

### CODE REVIEW

File: scenarios/traces/es-jaeger/Makefile [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	-helm repo add jaegertracing https://jaegertracing.github.io/helm-charts
+	helm repo update jaegertracing
+
+.PHONY: helm-dep
+helm-dep:
+	-helm dep build helm/
+
+.PHONY: namespace
+namespace:
+	kubectl create namespace "$(NS)"
```
Comment (paulfantom) [inline scenarios/traces/es-jaeger/Makefile:21]: Do we need a separate target when helm can do this for us with `--create-namespace`?

Comment (nhudson) [reply scenarios/traces/es-jaeger/Makefile:21]: Not anymore, I'll get that cleaned up.

File: stack/values.yaml [hunk 2]
```diff
@@ -118,3 +118,8 @@ kube-prometheus-stack:
         enabled: false
 opentelemetry-operator:
   enabled: true
+  kubeRBACProxy:
+    enabled: true
+    image:
+      repository: gcr.io/kubebuilder/kube-rbac-proxy
+      tag: v0.13.0
```
Comment (paulfantom) [inline stack/values.yaml:125]: Few questions:
- Why do we need it?
- Why not enable it by defult in tobs?
- Can we not specify image tag here? I fear it will go stale fairly quickly.

File: scenarios/traces/es-jaeger/helm/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  - https://github.com/timescale/helm-charts
+maintainers:
+  - name: timescale
+    url: https://www.timescale.com/
+keywords:
+  - observability
+  - monitoring
+  - tracing
+  - opentelemetry
+  - benchmarking
```
Comment (paulfantom) [inline scenarios/traces/es-jaeger/helm/Chart.yaml:16]: nit: those are optional fields and not needed if we just use `Chart.yaml` for depenedency management.

**Review by paulfantom — COMMENTED** [review]
Overall LGTM, just one question regarding kube-rbac-proxy.

File: stack/values.yaml (hunk 2 above)
Comment (nhudson) [reply stack/values.yaml:125]: Eh well not sure on this one.  Let me try again without it, but I was getting download errors on the version (`v0.11.0`).  Let me run through the scenario again without this and see what happens.  Could have been my wifi or something.

File: scenarios/traces/es-jaeger/helm/Chart.yaml (hunk 3 above)
Comment (nhudson) [reply scenarios/traces/es-jaeger/helm/Chart.yaml:16]: 👍 I'll clean it up.

**Review by paulfantom — APPROVED** [review]
